ipyleaflet/leaflet.py

```diff
@@ -846,6 +855,8 @@ class Map(DOMWidget, InteractMixin):
     default_style = InstanceDict(MapStyle).tag(sync=True, **widget_serialization)
     dragging_style = InstanceDict(MapStyle).tag(sync=True, **widget_serialization)
 
+    zoom_control = Bool(False)
```

Comment:
We should add observers on `zoom_control` so that when the user sets the attribute later on with:

```python
m.zoom_control = False
```

we remove the control from the map.

We can do that later.